src/ApiCore/ApiTransportInterface.php

```diff
@@ -47,16 +47,28 @@ public function startCall(Call $call, CallSettings $settings);
     /**
      * @param Call $call
      * @param CallSettings $settings The call settings to use for this call.
-     * @param string $streamingType
-     * @param string $resourcesGetMethod
+     * @param array $descriptor
+     *
+     * @return PromiseInterface
+     */
+    public function startOperationsCall(Call $call, CallSettings $settings, array $descriptor);
```

Comment:
Why do we want `startOperationsCall` and `getPagedListResponse` to be in the transport interface? I would expect their implementations to be the same between transports (and in fact that is the case, because they are implemented in the shared `ApiTransportTrait`).

Reply:
The thought is that if a user would like to implement their own transport, they know exactly which methods will be required in order to do so.

Reply:
Right. But, they **don't** need to implement those methods, because those methods are the same between transports. So if those methods are elsewhere in a transport-agnostic place, then they will be called correctly regardless of transport, right?

Reply:
After a discussion on this, reworked the transport interface a bit: https://github.com/googleapis/gax-php/pull/118/commits/aa93ff29d9374202ea6ed0e0ec75452ab9335cc4